system-tests/projects/screen-size/cypress/e2e/default_size.cy.js

```diff
@@ -16,7 +16,7 @@ describe('windowSize', () => {
         // availHeight: top.screen.availHeight,
       }).deep.eq({
         innerWidth: 1280,
-        innerHeight: 581, // chrome 128 decreased the size here from 633 to 581
+        innerHeight: 633, // chrome 119 increased the size here from 599 to 633
```

Comment:
did it increase again?

Reply:
No, passing `--disable-infobars` sets it back to what it was before (we probably should've set it back during the chrome 128 upgrade)

Reply:
I'm guessing it did this by default in a previous version?

Can we add a comment to that here just to make it clear why the size is what it is? Not sure if that gives all the context though

Reply:
It's a little more complex than that. We used to pass the flag but then chrome removed it so we removed it but they have since added it back. I'm not sure I can really add a comment here that will provide any meaningful context.